### PR TITLE
Remove old admin area

### PR DESCRIPTION
**Pull request by lancepioch**

### CODE REVIEW

File: routes/console.php
Comment (RMartinOscar) [inline routes/console.php:1]: Should we make a new override under `App\Console\Commands\Overrides\ServeCommand.php` ?

Comment (lancepioch) [reply routes/console.php:1]: Artisan Serve (PHP built in webserver) doesn't work with the React app we have, but we're replacing that and it works perfectly well with Filament.

File: app/Listeners/Auth/PasswordResetListener.php [hunk 1]
```diff
@@ -1,25 +0,0 @@
-<?php
```
[the remark this answers is not in the document]
Comment (lancepioch) [reply app/Listeners/Auth/PasswordResetListener.php:1]: It doesn't look to be referenced or used anywhere.

File: app/Listeners/Auth/TwoFactorListener.php [hunk 2]
```diff
@@ -1,17 +0,0 @@
-<?php
```
[the remark this answers is not in the document]
Comment (lancepioch) [reply app/Listeners/Auth/TwoFactorListener.php:1]: It doesn't look to be referenced or used anywhere.

File: resources/views/vendor/notifications/email.blade.php [hunk 3]
```diff
@@ -1,192 +0,0 @@
-<!DOCTYPE html>
```
[the remark this answers is not in the document]
Comment (lancepioch) [reply resources/views/vendor/notifications/email.blade.php:1]: It doesn't look to be referenced or used anywhere.

File: resources/views/vendor/notifications/email-plain.blade.php [hunk 4]
```diff
@@ -1,22 +0,0 @@
-<?php
```
[the remark this answers is not in the document]
Comment (lancepioch) [reply resources/views/vendor/notifications/email-plain.blade.php:1]: It doesn't look to be referenced or used anywhere.

File: resources/views/vendor/notifications/email.blade.php (hunk 3 above)
[the remark this answers is not in the document]
Comment (Boy132) [reply resources/views/vendor/notifications/email.blade.php:1]: Well, that's our email template. If we remove that we either get non working emails or emails in the default template (if such thing exists)

File: app/Listeners/Auth/AuthenticationListener.php [hunk 5]
```diff
@@ -1,30 +0,0 @@
-<?php
```
[the remark this answers is not in the document]
Comment (Boy132) [reply app/Listeners/Auth/AuthenticationListener.php:1]: image
image

[the remark this answers is not in the document]
Comment (lancepioch) [reply app/Listeners/Auth/AuthenticationListener.php:1]: That's very interesting, thanks, nice catch, I'll add it back!

[the remark this answers is not in the document]
Comment (Boy132) [reply app/Listeners/Auth/AuthenticationListener.php:1]: Yeah, you can't always trust phpstorms "no usages". Many things are used/ called indirectly by Laravel.
(fyi https://laravel.com/docs/11.x/events#defining-listeners)

**Comment by RMartinOscar** [comment]
`app/Exceptions/Solutions/ManifestDoesNotExistSolution.php`
and
https://github.com/pelican-dev/panel/blob/73f056c1c879ba896103709387a7d8f4c2fe8c50/composer.json#L53
should also get removed now that its not used anywhere.

**Comment by lancepioch** [comment]
@RMartinOscar ignition solutions are used in debug mode

**Review by Boy132 — APPROVED** [review]

File: resources/views/templates/wrapper.blade.php [hunk 6]
```diff
@@ -20,29 +20,22 @@
...
     </head>
     <body class="{{ $css['body'] ?? 'bg-neutral-50' }}">
         @section('content')
             @yield('above-container')
             @yield('container')
             @yield('below-container')
         @show
-        @section('scripts')
-            {!! $asset->js('main.js') !!}
-        @show
```
Comment (Boy132) [inline resources/views/templates/wrapper.blade.php:46]: This is needed for the client area.

File: app/Http/ViewComposers/AssetComposer.php [hunk 7]
```diff
@@ -2,24 +2,12 @@
...
-    public function __construct(private AssetHashService $assetHashService)
-    {
-    }
-
-    /**
-     * Provide access to the asset service in the views.
-     */
     public function compose(View $view): void
     {
-        $view->with('asset', $this->assetHashService);
```
Comment (Boy132) [inline app/Http/ViewComposers/AssetComposer.php:22]: Needed for the client area.

File: app/Services/Helpers/AssetHashService.php [hunk 8]
```diff
@@ -1,117 +0,0 @@
...
-        }
-
-        $manifest = static::$manifest;
-        if ($manifest === null) {
-            throw new ManifestDoesNotExistException();
-        }
-
-        return $manifest;
-    }
-}
```
Comment (Boy132) [inline app/Services/Helpers/AssetHashService.php:117]: Needed for the client area.

**Review by Boy132 — CHANGES_REQUESTED** [review]

**Comment by notAreYouScared** [comment]
The client ar
ea is working as expected. LGTM

@Boy132 @RMartinOscar Could you still take a look at it?

**Review by notAreYouScared — APPROVED** [review]

**Comment by RMartinOscar** [comment]
image
When clicking autoDeploy

**Review by Boy132 — APPROVED** [review]

**Pull request merged**